contracts/utils/Address.sol

```diff
@@ -59,7 +59,7 @@ library Address {
      * IMPORTANT: because control is transferred to `recipient`, care must be
      * taken to not create reentrancy vulnerabilities. Consider using
      * {ReentrancyGuard} or the
-     * https://solidity.readthedocs.io/en/v0.5.11/security-considerations.html#use-the-checks-effects-interactions-pattern[checks-effects-interactions pattern].
+     * https://solidity.readthedocs.io/en/latest/security-considerations.html#use-the-checks-effects-interactions-pattern[checks-effects-interactions pattern].
```

Comment:
Here we fixed the version because we can't be sure that this documentation will still be in "latest" in the future. 

Said otherwise:
- the content of latest can change
- the content of 0.5.11 cannot

Reply:
Let's fix it to `v0.8.0`.

We use `latest` in multiple parts of the code, and I don't think it's ok sending people to the `v0.5.11` docs